source/js/foundation/pages/youtube-regrets/categories-bar-chart.js

```diff
@@ -8,11 +8,11 @@ export const initYouTubeRegretsCategoriesBarChart = () => {
     gettext("VIOLENT OR GRAPHIC CONTENT"),
     gettext("COVID-19 MISINFORMATION"),
     gettext("HATE SPEECH"),
-    gettext("SPAM"),
+    gettext("SPAM, DECEPTIVE PRACTICES"),
     gettext("NUDITY & SEXUAL CONTENT"),
     gettext("CHILD SAFETY"),
     gettext("HARASSMENT & CYBERBULLYING"),
-    gettext("HARMFUL OR DANGEROUS"),
+    gettext("HARMFUL OR DANGEROUS CONTENT"),
```

Comment:
New content. @TheoChevalier you might need to know about this.

Reply:
Thanks for the heads up! Got translation going for these